utils/node_local_test.py

```diff
@@ -56,36 +56,39 @@ class NLTestTimeout(NLTestFail):
     """Used to indicate that an operation timed out"""
 
 
-instance_num = 0
+instance_num = 0  # pylint: disable=invalid-name
```

Comment:
Why not just rename?
```suggestion
INSTANCE_NUM= 0
```

Reply:
Because it's not a constant.  I was cheating a bit with suppressions this one but as it's only used in two places and they're right next to each other in the file I thought silencing was the easiest way.